pbtrader/hubot-headlines
# adding headlines script

### Description
Creating a hubot script to retrieve recent headlines from the New York Times. 

### JIRA Task Links
* https://akitabox.atlassian.net/browse/WEBAPP-4292

### Related PR Links
* None. Note that the AkitaBot package.json has not yet been updated with the new hubot script. 

### Post Deployment Checklist
- None

### Dev Owner Checklist
_Finish these before moving the JIRA task to `Ready for Review`_
- [ ] List manual testing workflows below
- [ ] Perform reviewer checklist on your changes 

### Manual Testing workflows
- [ ] List manual tests to cover changes/features/workflows not covered by automated tests
- Enter 'hubot headlines help' for help text
- Enter 'hubot headlines all' to see headlines from all NYTimes sections
- Enter 'hubot headlines [section]' to see headlines from a particular section

### Reviewer Checklist
_Finish these before merging this PR and moving the JIRA task to `Merged`_
- [ ] Read the changes looking for syntax errors, copypasta, and appropriate commenting
- [ ] Read the changes reviewing algorithms
    - [ ] New functionality is added with sound algorithms
    - [ ] Bugs are fixed using the appropriate solution given the time constraints and business priorities
- [ ] Ensure that automated test cases adequately cover the changes 
    - [ ] Tests to cover newly added functionality 
    - [ ] Tests to prevent the bug from being shipped and reproduced in the future
    - [ ] Are there any missing test cases that should be added?
- [ ] Manually test functionality not covered by automated tests (should be listed above in testing workflows)
    - [ ] Are there workflows that should be tested manually that are not listed?
- [ ] Does the finished product meet acceptance criteria listed in the applicable JIRA issue?
- [ ] **Are the changes production-ready and bug free?**
